changelogs/unreleased/gh-9556-fix-uri-ipv6.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/uri
```

Comment:
Nit: The first commit should be marked as `Part of #9556`.

Reply:
Done